tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -198,6 +217,31 @@ func runBugReportCommand(_ *cobra.Command, logOpts *log.Options) error {
 	return nil
 }
 
+func isDistroless(client kube.CLIClient, brConfig *config.BugReportConfig, ctx context.Context) (bool, error) {
+	iops, err := client.Dynamic().Resource(istioOperatorGVR).
+		Namespace(brConfig.IstioNamespace).
+		List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return false, fmt.Errorf("error while retrieving to IstioOperator CR - %v", err)
+	}
+
+	// TODO: WIP: Which iop to consider in case of multiple?
```

Comment:
I think if we have multiple control planes, we need to associate the proxy container with the control plane to determine if it's distroless?

Reply:
Added logic to get the effective IOP using `istiod` deployment. I am not sure it is the best possible approach to determine the effectiev IOP for the installation. Suggestions/Feedback would be really helpful here. Thanks.